manifests/cpp.yml

```
@@ -182,6 +182,8 @@ tests/:
     Test_Config_HttpClientErrorStatuses_FeatureFlagCustom: missing_feature
     Test_Config_HttpServerErrorStatuses_Default: missing_feature
     Test_Config_HttpServerErrorStatuses_FeatureFlagCustom: missing_feature
+    Test_Config_IntegrationEnabled_False: missing_feature
```

Can we look through the ci runs and confirm whether some of these tests are actually passing for some libraries.  It we be nice if we could add a descriptive message next to missing_feature 

ex: 
```suggestion
    # example reason, this is not accurate, please don't merge this suggestion
    Test_Config_IntegrationEnabled_False: missing_feature (integration name is confluent_kafka and not kafka)
```

---

I think that would be counter productive for this PR, I thought the goal was that the person owning the  language should test both **telemetry** and the related **config** test run and then enable it for that language, that way they would find what is missing for the config they may need to change and they can lead their language procces.